site/docs/guide-identity-brokering/00-guide-identity-brokering.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Identity Brokering
+tags: developer, organisation manager
+readingtime: 15min
+---
+
+<table class="table-wrapper">
+    <tr>
+        <td>Description</td>
+        <td>Learn about identity brokering/federation and how to add an external identity provider to authenticate your users.</td>
+    </tr>
+    <tr>
+        <td>Learning Outcomes</td>
+        <td>
+            In this module you will: 
+            <ul>
+                <li>Learn about Identity Providers</li>
+                <li>Add a new identity provider</li>
+                <li>Example with Google Login</li>
+            </ul>
+        </td>
+    </tr>
+     <tr>
+        <td>Prerequisites</td>
+        <td>
+            <ul>
+                <li>Knowledge of <a href="/zitadel-organizations">ZITADEL Organizations</a></li>
+            </ul>
+        </td>
+    </tr>
+</table>
+
+### What is Identity Brokering and Federated Identities?
+
+Federated identity management is an arrangement built upon the trust between two or more domains. Users of these domains are allowed to access applications and services using the same identity.
+This identity is known as federated identity and the pattern behind this as identity federation.
+
+A service provider that specializes in brokering access control between multiple service providers (also referred to as relying parties) is called identity broker.
+Federated identity management is an arrangement that is made between two or more such identity brokers across organizations.
+
+Example:
+If Google is configured as identity provider on your organization, the user will get the option to use his Google Account on the Login Screen of ZITADEL (1).
+ZITADEL will redirect the user to the login screen of Google where he as to authenticated himself (2) and is sent back after he has finished that (3). 
+Because Google is registered as trusted identity provider the user will be able to login in with the Google account after he linked an existing ZITADEL Account or just registered a new one with the claims provided by Google (4)(5).
+
+<figure itemprop="associatedMedia" itemscope itemtype="http://schema.org/ImageObject">
+    <a href="img/zitadel_identity_brokering.png" itemprop="contentUrl" data-size="1920x1080">
+        <img src="img/zitadel_identity_brokering.png" itemprop="thumbnail" alt="Zitadel Identity Brokering" />
+    </a>
+    <figcaption itemprop="caption description">Identity Brokering</figcaption>
+</figure>
+
+### Exercise: Register an external identity provider
+
+In this exercise we will add a new Google identity provider to federate identities with ZITADEL. 
+
+#### 1. Create new oAuth Client
+
+1. Register an OAuth Client in your preferred provider
+2. Make sure you add the ZITADEL callback redirect uris
+   https://accounts.zitadel.ch/register/externalidp/callback
+   https://accounts.zitadel.ch/login/externalidp/callback
+
+> **Information:** Make sure the provider is oAuth 2.0 compliant
```

Comment:
```suggestion
#### 1. Create new OIDC Client

1. Register an OIDC Client in your preferred provider
2. Make sure you add the ZITADEL callback redirect uris
   https://accounts.zitadel.ch/register/externalidp/callback
   https://accounts.zitadel.ch/login/externalidp/callback

> **Information:** Make sure the provider is OIDC 1.0 compliant with a proper Discovery Endpoint
```

Reply:
and i think it would help to either list the urls as entries or have them in a code block (```)